newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -239,6 +243,43 @@ const Instruction = (props: Props) => {
                 .getPlainString();
               expressionIsValid = resourcesManager.hasResource(resourceName);
             }
+            if (
+              expressionIsValid &&
+              parameterType === 'functionParameterName'
+            ) {
+              const eventsFunction = props.scope.eventsFunction;
```

Comment:
Could almoooost be a separate, tested function ^^ Not sure if it's worth it.

Reply:
Maybe an helper function should be in Core?